app/Http/Kernel.php

```diff
@@ -31,6 +31,7 @@ class Kernel extends HttpKernel
             \Rogue\Http\Middleware\EncryptCookies::class,
             \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
             \Illuminate\Session\Middleware\StartSession::class,
+            // \Illuminate\Session\Middleware\AuthenticateSession::class,
```

Comment:
Why is this commented out?

Reply:
It's how it is in the [framework code](https://github.com/laravel/laravel/blob/master/app/Http/Kernel.php#L36), as an option to comment back in and apply it. It didn't exist before in code so I added it to match the framework, but assumed it should stay commented out since it didn't exist before.

Reply:
It looks like this middleware is for invalidating sessions if a user's password changes (which we can't do with this middleware because passwords are stored in a separate service). If it's okay, let's remove this commented line so it doesn't look like it was commented out by mistake.

Reply:
Ok, sounds good. I'll make the same edit in Aurora then, since this change went through there but it has the same setup as Rogue to use a separate service. Thanks for the 👀  @DFurnes 😄